### PR TITLE
Joachim development

### PR DESCRIPTION
**Pull request by JoachimIsaac**
I modified the README.md fIle so that it would describe my repo more.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,9 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+# This repo contains [Joachim Isaac's](https://ssh.cloud.google.com/cloudshell/editor) portfolio and SPS projects.
```
Comment (stekolla) [inline README.md]: Maybe have this link to your Github or LinkedIn account instead of the cloud shell editor.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,9 @@
...
 
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/google/software-product-sprint.git`
-- Open the GitHub setup walkthrough: `teachme ~/software-product-sprint/walkthroughs/week-0-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
+Login to GitHub
+Clone this repo: by clicking the clone button
+If your using git through your terminal use: git clone https://github.com/YOUR-USERNAME/YOUR-REPOSITORY
```
Comment (stekolla) [inline README.md]: Update the URL to point to this repo: https://github.com/JoachimIsaac/software-product-sprint

Comment (stekolla) [inline README.md]: The first "your" should be "you're"

**Review by stekolla — CHANGES_REQUESTED** [review]
I've added a couple suggestions for the change

**Comment by JoachimIsaac** [comment]
I fixed the errors I had in the README file previously. Then I added links to my relevant sites(Github, LinkedIn, My Youtube Channel and my personal website). I also opted to add a banner at the top of the file.

File: README.md (hunk 1 above)
Comment (JoachimIsaac) [reply README.md]: Thank you I missed that, my bad.

**Review by stekolla — APPROVED** [review]
Looks good to me

**Pull request merged**